File: phyton/1134.py
a = int(input())
alc=0
gas=0
die=0

while a!=4:
    a = int(input())
    if a ==1:
        alc+=1
    elif a==2:
        gas+=1
    elif a==3:
        die+=1

print("MUITO OBRIGADO")
print("Alcool:",alc)
print("Gasolina:",gas)
print("Diesel:",die)

File: c/1134.c
#include <stdio.h>

int main (){
	int a, alc=0, gas=0, die=0;
	while (a != 4){
		scanf("%d",&a);
		if (a ==1){
			alc+=1;
		}else if (a == 2 ){
			gas +=1;
		}else if (a ==3){
			die +=1;
		}
	}
	printf("MUITO OBRIGADO\n");
	printf("Alcool: %d\n", alc);
	printf("Gasolina: %d\n", gas);
	printf("Diesel: %d\n", die);
	
	return 0;
}